ETL_pipeline/load_flood.py

```diff
@@ -0,0 +1,146 @@
+"""Load flood warnings into the database"""
+
+from os import environ as ENV
+
+import psycopg2
```

Comment:
Can we only import what we need, `from ... import ...`

Reply:
yep